.github/workflows/Downstream.yml

```
@@ -0,0 +1,56 @@
+name: Downstream
+
+on:
+  push:
+    branches: [master]
+    tags: [v*]
+  pull_request:
+
+jobs:
+  test:
+    name: ${{ matrix.package.repo }}/${{ matrix.package.group }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        julia-version: [1]
+        os: [ubuntu-latest]
+        package:
+          - {user: FluxML, repo: ObjectDetector.jl, group: All}
+          - {user: FluxML, repo: Transformers.jl, group: All}
+          - {user: FluxML, repo: GeometricFlux.jl, group: All}
+          - {user: FluxML, repo: FastAI.jl, group: All}
+          - {user: FluxML, repo: Flux3D.jl, group: All}
+          - {user: FluxML, repo: Torch.jl, group: All}
+          - {user: FluxML, repo: Metalhead.jl, group: All}
+          - {user: FluxML, repo: AtomicGraphNets.jl, group: All}
+
+    steps:
```

> but having it in every PR is maybe too much? Can we trigger it only on request same as people in Base do with@nanosoldier runtests(["JSON", "Crayons"], vs = ":master")?

This can be done by adding or removing a PR label, e.g., `run downstream test`.

```suggestion
    if: contains(github.event.pull_request.labels.*.name, 'run downstream test')
    steps:
```